.github/workflows/trufflehog.yml

```diff
@@ -0,0 +1,16 @@
+name: trufflehog
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  run-trufflehog:
+    name: Run trufflehog report
+    runs-on: ubuntu-latest
+    steps:
+     - uses: actions/checkout@master
```

Comment:
Does this have to user master instead of v1?

Reply:
good question - will look into it.

Reply:
```suggestion
     - uses: actions/checkout@v1
```